### PR TITLE
Gift Entry:  Exclude OCR groups

### PR DESCRIPTION
**Pull request by psadie-sfdo**
# Critical Changes

# Changes
- Exclude OCR Object Mappings from available form field selections in Template Builder
# Issues Closed

# Community Ideas Delivered

# New Metadata

# Deleted Metadata

### CODE REVIEW

File: src/lwc/geTemplateBuilderFormFields/geTemplateBuilderFormFields.js [hunk 1]
```diff
@@ -27,6 +27,11 @@ const REQUIRED_FORM_FIELDS = [
     CONTACT1_LASTNAME_INFO.fieldApiName
 ];
 
+const EXCLUDED_OBJECT_MAPPINGS = [
+    'Opportunity_Contact_Role_1_',
```
Comment (ptewson-sfdo) [inline src/lwc/geTemplateBuilderFormFields/geTemplateBuilderFormFields.js]: This is a minor detail, but could you leave off the trailing underscore here and change the math on the substring below to be 'length - 10'? It both looks nicer, and would match to one of our default, protected mappings if for whatever reason we ever had those in play

**Comment by annejackson** [comment]
Do we need to lurch the WI for this @psadie-sfdo ?

**Comment by psadie-sfdo** [comment]
**lurch: attach W-038983

**Comment by LurchTheButler** [comment]
Tracking <a href="https://foundation.my.salesforce.com/a2x1E000001HrITQA0" target="_blank">W-038983</a>

**Pull request merged**